src/main/java/net/imglib2/algorithm/morphology/table2d/Spur.java

```diff
@@ -73,7 +73,7 @@
 
 		final ImgFactory< T > factory;
 		if ( extendedVal instanceof NativeType )
-			factory = ( ImgFactory< T > ) Util.getArrayOrCellImgFactory( target, ( NativeType ) extendedVal );
+			factory = Util.< NativeType >getArrayOrCellImgFactory( target, ( NativeType ) extendedVal );
```

Comment:
As a side note: Closing IDEA and re-opening the project would trigger the same error. "Re-writing" to the exact same code would make the error go away.